R/join.R

```diff
@@ -0,0 +1,89 @@
+#' @ref{https://gist.github.com/jimhester/a060323a05b40c6ada34}
+#' @export
+insensitive_join <- function(fun = dplyr::left_join, type = "LEFT") {
```

Comment:
by or other args are not defined.

Reply:
this is helper function which changes the behavior of the join function so argument by or other arguments are not required. as you can see below, by and other arguments are handled inside `substitute` like below.

```
body(new_fun) <- substitute({
    by <- dplyr:::common_by(by, x, y)
``